step-plans/step-plans-base-artefacts/src/main/java/step/artefacts/TestSet.java

```diff
@@ -22,7 +22,7 @@
 import step.core.artefacts.Artefact;
 import step.core.dynamicbeans.DynamicValue;
 
-@Artefact(name = "TestSet", useAsTemplate = true)
+@Artefact(name = "TestSet", validAsRoot = true, validAsControl = false)

```

Comment:
What happens for existing plans containing TestSet as child? If I want to have some initialization (i.e. Set controls, before starting the Test and parallel executions of its test cases, I cannot do it anymore?

Reply:
You're right. Reverting